### PR TITLE
Fix typo with js graphql documentation

### PR DESCRIPTION
**Pull request by chowerth**
Grammatical changes explaining graphql mutations. Changed query to mutation for example code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mauerbac** [comment]
Pinged JS team for review

File: docs/lib/graphqlapi/fragments/js/mutate-data.md [hunk 1]
```diff
@@ -62,15 +62,15 @@ const todoDetails = {
 };
 
 const todo = await API.graphql({
-  query: queries.createTodo,
+  mutation: mutations.createTodo,
```
Comment (elorzafe) [inline docs/lib/graphqlapi/fragments/js/mutate-data.md:65]: This should be
```suggestion
  query: mutations.createTodo,
```

**Comment by mauerbac** [comment]
ping @chowerth

**Review by sw-yx — APPROVED** [review]

**Review by mauerbac — APPROVED** [review]

**Review by elorzafe — APPROVED** [review]
👍

**Pull request merged**